tests/e2e/pageObjects/add-redis-database-page.ts

```diff
@@ -7,176 +7,242 @@ export class AddRedisDatabasePage {
+        this.redisClusterType = Selector('[data-test-subj=radio-btn-enterprise-cluster]');
+        this.redisCloudProType = Selector('[data-test-subj=radio-btn-cloud-pro]');
+        this.redisSentinelType = Selector('[data-test-subj=radio-btn-sentinel]');
+        this.showDatabasesButton = Selector('[data-testid=btn-show-databases]');
+        this.databaseName = Selector('.euiTableCellContent.column_name');
+        this.selectAllCheckbox = Selector('[data-test-subj=checkboxSelectAll]');
+        this.databaseIndexCheckbox = Selector('[data-testid=showDb]~div');
+        //TEXT INPUTS (also referred to as 'Text fields')
+        this.hostInput = Selector('[data-testid=host]');
+        this.portInput = Selector('[data-testid=port]');
+        this.databaseAliasInput = Selector('[data-testid=name]');
+        this.passwordInput = Selector('[data-testid=password]');
+        this.usernameInput = Selector('[data-testid=username]');
+        this.discoverSentinelDatabaseButton = Selector('[data-testid=btn-submit]');
+        this.accessKeyInput = Selector('[data-testid=access-key]');
+        this.secretKeyInput = Selector('[data-testid=secret-key]');
+        this.welcomePageTitle = Selector('[data-testid=welcome-page-title]');
+        this.databaseIndexInput = Selector('[data-testid=db]');
+        this.errorMessage = Selector('[data-test-subj=toast-error]');
+    }
 
-  /**
-   * Adding a new redis database with index
-   * @param parameters the parameters of the database
-   * @param index the logical index of database
-   */
-   async addLogicalRedisDatabase(parameters: AddNewDatabaseParameters, index: string): Promise<void> {
-    await t
-        .click(this.addDatabaseButton)
-        .click(this.addDatabaseManually)
-    await t
-        .typeText(this.hostInput, parameters.host, { replace: true, paste: true })
-        .typeText(this.portInput, parameters.port, { replace: true, paste: true })
-        .typeText(this.databaseAliasInput, parameters.databaseName, { replace: true, paste: true })
-    if (!!parameters.databaseUsername) {
-        await t.typeText(this.usernameInput, parameters.databaseUsername, { replace: true, paste: true })
+    /**
+     * Adding a new redis database
+     * @param parameters the parameters of the database
+     */
+    async addRedisDataBase(parameters: AddNewDatabaseParameters): Promise<void> {
+        await t
+            .click(this.addDatabaseButton)
+            .click(this.addDatabaseManually)
+        await t
+            .typeText(this.hostInput, parameters.host, {
+                replace: true,
+                paste: true
+            })
+            .typeText(this.portInput, parameters.port, {
+                replace: true,
+                paste: true
+            })
+            .typeText(this.databaseAliasInput, parameters.databaseName, {
+                replace: true,
+                paste: true
+            })
```

Comment:
Whi you did it? please revert

Reply:
It was added automatically by IDE when I corrected all spaces in the file. Fixed.